include/packetgraph/vtep.h

```diff
@@ -46,6 +46,10 @@ struct ether_addr;
  */
 struct ether_addr *pg_vtep_get_mac(const struct pg_brick *brick);
 
+void pg_vtep4_cleanup_mac(struct pg_brick *brick, int port_id);
+void pg_vtep6_cleanup_mac(struct pg_brick *brick, int port_id);
```

Comment:
Add somes descriptions for doxygen documentation
@param description
@param description